beacon-chain/node/node.go

```diff
@@ -214,8 +214,22 @@ func (b *BeaconNode) startDB(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	if clearDB || forceClearDB {
-		d, err = confirmDelete(d, dbPath, forceClearDB)
+	clearDBConfirmed := false
+	if clearDB && !forceClearDB {
+		actionText := "This will delete your beacon chain data base stored in your data directory. " +
```

Comment:
there isnt any need to to pass in the text to the method. The method can log it in there itself

Reply:
nvm , it seems like this is a generic function , you can ignore the comment then